03_launch_mgmt_cluster.sh

```diff
@@ -79,6 +79,20 @@ function launch_minikube() {
     minikube start
 }
 
+function launch_kinder() {
+  kinder="${GOPATH}/bin/kinder"
```

Comment:
shouldn't this already be in `$PATH` ?

Reply:
Yes, but not in $PATH of root user. Therefore, I use the full path to run Kinder with sudo right.

Reply:
ah, OK, makes sense.